sample/build.gradle

```diff
@@ -33,27 +28,26 @@ android {
 dependencies {
 
     // Android
-    compile 'com.android.support:appcompat-v7:23.1.1'
-    compile 'com.android.support:recyclerview-v7:23.1.1'
-    compile 'com.android.support:cardview-v7:23.1.1'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.android.support:recyclerview-v7:28.0.0'
+    implementation 'com.android.support:cardview-v7:28.0.0'
 
     // Heimdall
-    compile project(':library')
+    implementation project(':library')
 
     // Rx
-    compile 'io.reactivex:rxandroid:1.1.0'
-    compile 'io.reactivex:rxjava:1.1.0'
-    compile 'com.trello:rxlifecycle:0.4.0'
-    compile 'com.trello:rxlifecycle-components:0.4.0'
+    implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
+    implementation 'io.reactivex.rxjava2:rxjava:2.2.7'
+    implementation 'com.trello.rxlifecycle3:rxlifecycle-components:3.0.0'
 
     // Serialization
-    compile 'com.google.code.gson:gson:2.4'
-    compile 'org.parceler:parceler-api:0.2.16'
+    implementation 'com.google.code.gson:gson:2.4'
```

Comment:
This is updated in the library, so I guess it makes sense to update it here as well.

Reply:
sure :) 👍